lib/spack/spack/compilers/clang.py

```diff
@@ -157,6 +157,20 @@ def cxx17_flag(self):
             else:
                 return "-std=c++17"
 
+    @property
+    def c99_flag(self):
+        return '-std=c99'
+
+    @property
+    def c11_flag(self):
+        if self.version < ver('6.1.0'):
+            raise UnsupportedCompilerFlag(self,
+                                          "the C11 standard",
+                                          "c11_flag",
+                                          "< 3.3")
```

Comment:
Version mismatch? The check is for `6.1.0` but the exception message says `3.3`.

Reply:
Oops, my mistake. I opened another PR #11632 with fixes.